ddl/ddl_worker.go

```diff
@@ -282,41 +293,56 @@ func (w *worker) finishDDLJob(t *meta.Meta, job *model.Job) (err error) {
 	return errors.Trace(err)
 }
 
+func isDependencyJobDone(t *meta.Meta, job *model.Job) (bool, error) {
```

Comment:
Will there be multiple job DDL dependencies?

Reply:
We only record the maximum job ID in multiple dependent jobs.